checker/tests/units/PolyUnitTest.java

```diff
@@ -4,34 +4,33 @@
 
 public class PolyUnitTest {
 
-  @PolyUnit int triplePolyUnit(@PolyUnit int amount) {
-    return 3*amount;
-  }
+    private @PolyUnit int triplePolyUnit(@PolyUnit int amount) {
```

Comment:
Visibility modifiers just add noise to test cases, you should remove them again. The other whitespace changes should also be undone. (If you feel strongly about the whitespace, make a separate pull request just for that.)